lib/sanitize.js

```diff
@@ -111,3 +123,12 @@ function sanitizeCellStyle (tagName, attribs) {
   var result = alignmentRegEx.exec(attribs.style || '')
   return result ? cell(result[1]) : cell()
 }
+
+function prefixHTMLids (tagName, attribs) {
+  if (attribs.id && attribs.id.replace('user-content-', '').length > 0) {
```

Comment:
basically the idea is that we need to check that we didn't already namespace the user content id during the marky parse